eslint.config.js

```diff
@@ -139,7 +139,9 @@ module.exports = [
       'jsdoc/require-returns-type': 'off',
       // Ensure there is a blank line between the body and any @tags,
       // as required by the tsdoc spec.
-      'jsdoc/tag-lines': ['error', 'any', {'startLines': 1}],
+      // TODO: Re-enable soon.
```

Comment:
Do you want to file a separate issue for this puppy? (and maybe ones for the other future work you mentioned in the PR description)